packages/esm-home-app/src/homepage-metrics/TestDataTile.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Tile, Layer } from '@carbon/react';
+import styles from './home-metrics-widget.scss';
+import { ArrowRight } from '@carbon/react/icons';
+
+interface DataTileProps {}
+
+const DataTile = () => {
+  const dataValue = 'default';
+  return (
+    <React.Fragment>
+      <Layer className={styles.container}>
+        <Tile className={styles.tileContainer}>
+          <div>
+            <h2>Data to display: {dataValue}</h2>
+          </div>
```

Comment:
wrapping of `h2` by `div` has no significance.  You can put `h2` directly under `Title` element.